tutor/commands/config.py

```diff
@@ -255,9 +259,41 @@ def patches_show(context: Context, name: str) -> None:
         print(rendered)
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context) -> None:
+    config_file = tutor_config.config_path(context.root)
+
+    if not os.path.isfile(config_file):
+        raise exceptions.TutorError(
+            f"Missing config file at {config_file}. Have you run 'tutor local launch' yet?"
+        )
+
+    open_cmd = None
+
+    if which("open"):  # MacOS & linux distributions that ship `open`. eg., Ubuntu
+        open_cmd = ["open", config_file]
+    elif which("xdg-open"):  # Linux
+        open_cmd = ["xdg-open", config_file]
+    elif which("start"):  # Windows
+        # Calling "start" on a regular file opens it with the default editor.
+        # The second argument "" just means "don't give the window a custom title".
+        open_cmd = ["start", '""', config_file]
+    else:
+        raise exceptions.TutorError(
+            f"Failed to find utility to launch an editor. Edit {config_file} with the editor of your choice."
+        )
+
+    subprocess.call(open_cmd)
```

Comment:
For CLI UX consistency, please use the `execute` function from `tutor.utils`. It is the same as this line, except it prints out the underlying command in purple for the user's benefit, and it formats any errors in red instead of showing a stack trace.